services/brig/test/integration/API/MLS.hs

```diff
@@ -0,0 +1,126 @@
+module API.MLS where
+
+import Bilge
+import Bilge.Assert
+import Brig.Options
+import Data.ByteString.Conversion
+import qualified Data.ByteString.Lazy as LBS
+import Data.Domain
+import Data.Id
+import Data.Qualified
+import qualified Data.Set as Set
+import qualified Data.Text as T
+import Imports
+import System.Process
+import Test.Tasty
+import Test.Tasty.HUnit
+import UnliftIO.Temporary
+import Util
+import Wire.API.MLS.KeyPackage
+import Wire.API.User
+import Wire.API.User.Client
+
+tests :: Manager -> Brig -> Opts -> TestTree
+tests m b _opts =
+  testGroup
+    "MLS"
+    [ test m "POST /mls/key-packages/self/:client" (testKeyPackageUpload b),
+      test m "GET /mls/key-packages/self/:client/count" (testKeyPackageZeroCount b),
+      test m "GET /mls/key-packages/claim/:domain/:user" (testKeyPackageClaim b)
+    ]
+
+testKeyPackageUpload :: Brig -> Http ()
+testKeyPackageUpload brig = do
+  u <- userQualifiedId <$> randomUser brig
```

Comment:
There's also the `randomQualifiedUser` function in case you find it nice to use.

Reply:
That's in galley, isn't it?

Reply:
Oh, I wasn't aware where the function is coming from! My apologies.